babel.config.js

```diff
@@ -0,0 +1,13 @@
+/* eslint linebreak-style: ["error", "windows"] */
```

Comment:
Hey @mthamayor , I don't think this line is necessary here. You may want to remove it. I believe that is why the hound bot is giving you an error about linebreaks.

Reply:
Hello, I have worked on it as requested